course_info/static/course_info/js/controllers/SearchController.js

```diff
@@ -105,12 +105,24 @@
                 });
             };
 
+            $scope.getCourseDescription = function(course) {
+                // If a course's title is [NULL], attempt to display the short title.
+                // If the short title is also [NULL], display [School] 'Untitled Course' [Term Display]
+                if(typeof course.title != "undefined" && course.title.length > 0){
+                    return course.title;
+                }
+                else if(typeof course.short_title != "undefined" && course.short_title.length > 0){
+                    return course.short_title;
+                }
+                return 'Untitled Course';
```

Comment:
The way I read the AC, we need to prepend the school and append the term display, is that not the way it was meant?